src/js/component.js

```diff
@@ -1553,22 +1572,22 @@ class Component {
     }
 
     if (name === 'Player' && Component.components_[name]) {
-      const Player = Component.components_[name];
+      const players = Component.components_[name].players;
```

Comment:
It's not, but it's checked above in line 1574. I guess that could be clearer. Will update.